app/kumactl/cmd/install/install_transparent_proxy.go

```diff
@@ -158,7 +163,8 @@ runuser -u kuma-dp -- \
 	cmd.Flags().StringVar(&args.ExcludeOutboundPorts, "exclude-outbound-ports", args.ExcludeOutboundPorts, "a comma separated list of outbound ports to exclude from redirect to Envoy")
 	cmd.Flags().StringVar(&args.User, "kuma-dp-user", args.UID, "the user that will run kuma-dp")
 	cmd.Flags().StringVar(&args.UID, "kuma-dp-uid", args.UID, "the UID of the user that will run kuma-dp")
-	cmd.Flags().BoolVar(&args.RedirectDNS, "redirect-dns", args.RedirectDNS, "redirect the DNS requests to a specified port")
+	cmd.Flags().BoolVar(&args.RedirectDNS, "redirect-dns", args.RedirectDNS, "redirect all DNS requests to the servers in /etc/resolv.conf to a specified port")
+	cmd.Flags().BoolVar(&args.RedirectAllDNSTraffic, "redirect-all-dns-traffic", args.RedirectAllDNSTraffic, "redirect all DNS requests to a specified port")
```

Comment:
If I had found `--redirect-all-dns-traffic` flag in the docs and I try to use it then it wouldn't be really clear that I also have to use `--redirect-dns` as well to make it work. Maybe we can make it mutually exclusive? So it could be `--redirect-dns-resolv-conf` and `--redirect-dns-all-traffic`, what do you think?

Reply:
Makes sense, let me see what I can do about it.